examples/LaTeX/Dop.py

```diff
@@ -10,7 +10,7 @@
 f = o3d.mv('f', 'scalar', f=True)
 lap = o3d.grad*o3d.grad
 print(r'\nabla =', o3d.grad)
-print(r'%\nabla^{2} = \nabla . \nabla =', lap)
+print(r'%\nabla^{2} = \nabla \cdot \nabla =', lap)
```

Comment:
The percent is supposed to mean "do not postprocess this", as far as I can tell.

At any rate, this line already contains latex, so there's no point translating only bits of it.